spec/features/movies/show_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe "Movie Show Page" do
+  let!(:studio1) {Studio.create!(name: "Univeral Studios", location: "Hollywood")}
+  
+  let!(:matrix) {studio1.movies.create!(title: "The Matrix", creation_year: 1999, genre: "Action")}
+  let!(:speed) {studio1.movies.create!(title: "Speed", creation_year: 1994, genre: "Action")}
+  let!(:always) {studio1.movies.create!(title: "Always Be My Maybe", creation_year: 2019, genre: "Rom Com")}
+
+  let!(:keanu) {Actor.create!(name: "Keanu Reeves", age: 58)}
+  let!(:laurence) {Actor.create!(name: "Laurence Fishburne", age: 61)}
+  let!(:carrie) {Actor.create!(name: "Carrie-Anne Moss", age: 55)}
+  let!(:sandra) {Actor.create!(name: "Sandra Bullock", age: 58)}
+  let!(:ali) {Actor.create!(name: "Ali Wong", age: 41)}
+
+  describe "US2 I see the movie's title, creation year, and genre," do
+    it "I see a list of all its actors from youngest to oldest. And I see the average age of all of the movie's actors" do
```

Comment:
This is a really robust test! Nice assertions. You could probably break it up into 2 different test blocks (it can display movie attributes, and then it can display all actors youngest to oldest)